common/api/ecschema-locaters.api.md

```diff
@@ -43,6 +43,12 @@ export abstract class SchemaFileLocater {
     searchPaths: string[];
 }
 
+// @beta
+export class SchemaFileUtility {
```

Comment:
This looks like it's supposed to be a `namespace`, not a `class`. The word "utility" in a type name usually indicates the type is just a grab-bag container. Consider just exporting two free functions or, if you must keep them in a container object, give the container a better name.

Reply:
done